### PR TITLE
Add Kraken Trading Pairs sync

### PR DESCRIPTION
**Pull request by mortezaalizadeh**

### CODE REVIEW

File: src/exchange/kraken-processor/config.yaml [hunk 1]
```diff
@@ -16,6 +16,8 @@ postgres:
   maxOpenConns: 5
 
 kraken:
+  apiKey: ""
+  secretKey: ""
   orderBook:
     pairs:
       - pair: XBT/USD
```
Comment (sepisoad) [inline src/exchange/kraken-processor/config.yaml:23]: why do we have hardcoded pairs in the config,
do we use them anywhere?

File: src/exchange/kraken-processor/subscribers/tradingpairs-subscriber.go [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+	"github.com/aopoltorzhicky/go_kraken/rest"
+	"github.com/omiga-group/omiga/src/exchange/kraken-processor/configuration"
+	"github.com/omiga-group/omiga/src/exchange/kraken-processor/mappers"
+	exchangeConfiguration "github.com/omiga-group/omiga/src/exchange/shared/configuration"
+	"github.com/omiga-group/omiga/src/exchange/shared/repositories"
+	"github.com/omiga-group/omiga/src/shared/enterprise/cron"
+	"go.uber.org/zap"
+)
+
+type KrakenTradingPairsSubscriber interface {
```
Comment (sepisoad) [inline src/exchange/kraken-processor/subscribers/tradingpairs-subscriber.go:15]: did you intentionally left this interface as empty,
it looks that `Run` method could go into this interface, right!

or is this used for mock codegen

File: src/exchange/kraken-processor/mappers/order-book.go [hunk 3]
```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/omiga-group/omiga/src/shared/enterprise/decimal"
 )
 
-func FromKrakenOrderBookToOrderBook(
+func KrakenOrderBookToOrderBook(
```
Comment (sepisoad) [inline src/exchange/kraken-processor/mappers/order-book.go:14]: yes

**Review by sepisoad — CHANGES_REQUESTED** [review]
added a small comment

File: src/exchange/kraken-processor/subscribers/tradingpairs-subscriber.go (hunk 2 above)
Comment (mortezaalizadeh) [reply src/exchange/kraken-processor/subscribers/tradingpairs-subscriber.go:15]: it is used for Wire to inject dependencies, the RUN method is the internal method called by the scheduler. If we ever wanna let others to invoke the RUN for whatever reason outside what it is scheduler for, we can then add it to the interface @sepisoad

**Pull request merged**